components/DeleteRecipe.js

```diff
@@ -0,0 +1,21 @@
+import TrashBin from "public/svg/TrashBin.svg";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+export default function DeleteRecipe({ onDeleteRecipe, recipeId }) {
+  const router = useRouter();
+
+  function handleDeleteAndReroute() {
+    if (confirm("Are you sure?")) {
```

Comment:
```suggestion
    if (confirm("Are you sure you want to delete this recipe?")) {
```

Maybe you want to be more specific here, so the user really knows what will happen when he confirms after clicking the icon.